core/src/main/groovy/noe/ews/utils/JwsNameHelper.groovy

```diff
@@ -122,4 +124,12 @@ class JwsNameHelper {
   void toggleCompatibilityMode() {
     this.compatibilityMode = !compatibilityMode
   }
+
+  /**
+   * Sets architecture separator so that you can get old/new naming scheme of the zipnames.
+   * Architecture separator is used only for Windows and Solaris packages.
+   */
+  void setArchSeparator(String separator) {
```

Comment:
What is the benefit of this method, since groovy generates setter and getter automatically?

Reply:
I can remove it....

Reply:
Why you wouldn't if it has no sense ?